client/my-sites/stats/post-trends/day.jsx

```diff
@@ -35,58 +41,45 @@ module.exports = React.createClass( {
 		this.setState( { showPopover: false } );
 	},
 
-	buildTooltipData: function( date, data ) {
-		var tooltipData = [],
-			count = 0,
-			label,
-			labelWrapper;
-
-		count = data[ date.format( 'YYYY-MM-DD' ) ] || 0;
-
-		label = this.translate(
-			'%(posts)d post on',
-			'%(posts)d posts on', {
-				count: count,
-				args: {
-					posts: count
-				},
-				comment: 'How many posts published on a certain date.'
-		} );
-
-		labelWrapper = (
+	buildTooltipData: function() {
+		const { date, postCount } = this.props;
+		const count = postCount || 0;
+
+		const labelWrapper = (
 			<span>
-				<span className="post-count">{ label } </span>
+				<span className="post-count">{
+					this.translate(
```

Comment:
I know this existed previously, but I can't help but think this isn't very translatable, especially for languages which might insert the date somewhere in the middle of the sentence. Any reason we couldn't add the date as an argument, especially with component interpolation?

```jsx
this.translate( 'Post on {{dateWrapper}}%s{{/dateWrapper}}, { args: [ date.format( 'L' ) ], components: { dateWrapper: <span className="date" /> } } )
```